tanushree-sharma/fydp-app
# Adding underline on the nav bar

Testing:
- click on the different nav bar options to make sure underline is showing up only on the active item
- click the logo to make sure it redirects to the home page
